OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -1562,6 +1562,49 @@ public void TestIsFeatureEnabledGivenVariationNotFoundInFeatureExperimentButInRo
             LoggerMock.Verify(l => l.Log(LogLevel.INFO, "Feature flag \"boolean_single_variable_feature\" is enabled for user \"testUserId\"."), Times.Once);
         }
 
+        public void TestIsFeatureEnabledWithFeatureEnabledPropertyGivenFeatureExperiment()
+        {
+            var userId = "testUserId2";
+            var featureKey = "double_single_variable_feature";
+            var experiment = Config.GetExperimentFromKey("test_experiment_double_feature");
+            var featureEnabledTrue = Config.GetVariationFromKey("test_experiment_double_feature", "control");
+            var featureEnabledFalse = Config.GetVariationFromKey("test_experiment_double_feature", "variation");
+            var featureFlag = Config.GetFeatureFlagFromKey(featureKey);
+            var decisionTrue = new FeatureDecision(experiment, featureEnabledTrue, FeatureDecision.DECISION_SOURCE_EXPERIMENT);
+            var decisionFalse = new FeatureDecision(experiment, featureEnabledFalse, FeatureDecision.DECISION_SOURCE_EXPERIMENT);
+
+            DecisionServiceMock.Setup(ds => ds.GetVariationForFeature(featureFlag, TestUserId, null)).Returns(decisionTrue);
+            DecisionServiceMock.Setup(ds => ds.GetVariationForFeature(featureFlag, userId, null)).Returns(decisionFalse);
+
+            var optly = Helper.CreatePrivateOptimizely();
+            optly.SetFieldOrProperty("DecisionService", DecisionServiceMock.Object);
+
+            // Verify that IsFeatureEnabled returns true when feature experiment variation's 'featureEnabled' property is true.
+            bool result = (bool)optly.Invoke("IsFeatureEnabled", featureKey, TestUserId, null);
+            Assert.True(result);
+
+            // Verify that IsFeatureEnabled returns false when feature experiment variation's 'featureEnabled' property is false.
+            result = (bool)optly.Invoke("IsFeatureEnabled", featureKey, userId, null);
+            Assert.False(result);
```

Comment:
Why does it return false? I believe that you should also mock getVariationForFeatureRollout as well, what if getVariationForFeatureRollout returns any variation.